rfcs/load_test.md

```diff
@@ -0,0 +1,46 @@
+# RFC 33: Crash Tests
+
+## Summary
+
+Add a test type in which the test document is simply loaded. The test
+passes if this occurs without encountering a fatal error e.g. a crash.
```

Comment:
Does it pass if it encounters a non-fatal error, such as a leak?

Reply:
I think that's a browser-specific consideration since leak checking is a browser-specific feature at this point (for gecko at least we can only detect leaks per browser invocation, not per test, so they have to be handled separately).